src/language/languageServiceHost.ts

```diff
@@ -30,8 +30,7 @@ module Lint {
     }
 
     export function createLanguageService(fileName: string, source: string) {
-        const documentRegistry = ts.createDocumentRegistry();
```

Comment:
Doing this ourselves was unnecessary as it is done internally automatically by the language service